### PR TITLE
Move related commands as subcommands into /privacy

### PR DESCRIPTION
**Pull request by jubalh**
Fix https://github.com/profanity-im/profanity/issues/1836

### CODE REVIEW

**Comment by jubalh** [comment]
@H3rnand3zzz could you review?

**Comment by jubalh** [comment]
I added some more changes. Will have to self test myself tomorrow :)

File: src/ui/console.c [hunk 1]
```diff
@@ -2934,22 +2934,38 @@ cons_strophe_setting(void)
...
     } else {
         cons_show("Reveal OS name when asked for software version (XEP-0092) (/privacy os)    : OFF");
     }
+
+    if (connection_get_status() == JABBER_CONNECTED) {
+        char* account_name = session_get_account_name();
+        ProfAccount* account = accounts_get_account(account_name);
+
+        if (account->client) {
+            cons_show("Client name (/account set <account> clientid)  : %s", account->client);
```
Comment (H3rnand3zzz) [inline src/ui/console.c:2942]: Hopefully, the spacing can be fixed.

**Review by H3rnand3zzz — CHANGES_REQUESTED** [review]

File: src/ui/console.c (hunk 1 above)
Comment (jubalh) [reply src/ui/console.c]: should be done

**Review by H3rnand3zzz — APPROVED** [review]

**Pull request merged**